### PR TITLE
Export event logs to csv

### PR DESCRIPTION
**Pull request by inbarbarkai**
Implemented export of event logs to csv.

Described in issue #31.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,6 +29,8 @@ services:
       - POSTGRES_PASSWORD=password
       - POSTGRES_USER=invitease
       - POSTGRES_DB=invitease
+    ports:
```
Comment (yarons) [inline docker-compose.yml]: I'll remove that in favor of `dbadmin` docker.

File: src/core/utilities.py [hunk 2]
```diff
@@ -10,7 +11,13 @@ def create_users_repository():
...
 
+
 def create_eventlog_repository():
     connection = create_connection()
     return EventLogRepository(connection)
 
+
+def translate_export_headers(headers: list, model) -> dict:
+    result = [(x, y) for x, y in model.__dict__.items() if x in headers]
+    result = {x: y.info.get('export_title', x) for x, y in result}
```
Comment (yarons) [inline src/core/utilities.py:22]: There's the `.addheader` function, will it add an additional header or it replaces the header to include?

**Comment by inbarbarkai** [comment]
Sure, I don't care what language we use

**Review by yarons — APPROVED** [review]
Added some changes of my own and rebased.

**Pull request merged**